default.nix

```diff
@@ -141,6 +142,15 @@ in rec {
 
       npmFlags = npmFlagsNpm;
       buildCommand = ''
+        # Inside nix-build sandbox $HOME points to a non-existing
+        # directory, but npm may try to create this directory (e.g.
+        # when you run `npm install` or `npm prune`) and will succeed
+        # if you have a single-user nix installation (because / is
+        # writable in this case), causing different behavior for
+        # single-user and multi-user nix. Create read-only $HOME to
+        # prevent it
+        mkdir -p --mode=a-w "$HOME"
```

Comment:
Wait, what? Are you saying that single-user Nix with sandboxing enabled has writeable `/`? Isn’t that a bug? Why would anyone want this?

Reply:
I think on multi-user nix `/` is protected by having a different owner, so maybe they didn't consider it important enough to implement another logic just for a single-user mode